x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/metrics.tsx

```diff
@@ -124,10 +124,11 @@ const TabComponent = (props: TabProps) => {
     region,
     false,
     {
-      interval: '1m',
```

Comment:
Are the changes to this file intentional?

Reply:
Whoops... I think I was trouble shooting on this PR and forgot to change that back.